charts/zwavejs2mqtt/1.3.0/questions.yaml

```diff
@@ -78,6 +78,30 @@ questions:
       $ref:
         - "definitions/timezone"
 
+  - variable: PUID
+    group: "Configuration"
+    label: "PUID"
+    description: "The UserID of the user running the application and owning the files"
+    schema:
+      type: int
+      default: 568
+
+  - variable: PGID
+    group: "Configuration"
+    label: "PGID"
+    description: "The groupID of the user/group running the application and owning the files"
+    schema:
+      type: int
+      default: 568
+
+  - variable: UMASK
+    group: "Configuration"
+    label: "UMASK (advanced)"
+    description: "The UMASK used if supported by the application"
+    schema:
+      type: string
+      default: "002"
+
```

Comment:
I don't think this is supported - It's also possible I just don't understand how this works, or what's it's used for.

Reply:
(See above. )
Please report any actual bugs with it, but not likely to be removed as it serves a multitude of purposes.